samples/daily-prebuilt/permissions-can-admin/src/components/AdminPanel/AdminPanel.js

```diff
@@ -4,24 +4,27 @@ const ParticipantListItem = ({
   p,
   makeAdmin,
   removeFromCall,
-  isOwner,
+  localIsOwner,
```

Comment:
I was thinking renaming this to have `local` might make it clearer that this attribute is specific to the local user, _not_ the participant being passed in